### DISCUSSION TITLE
Global Path Annotation in Interface ?

### DISCUSSION THREAD

**Discussion by soumyadeep-ghosh** [square/retrofit#3548] [General]
Well I searched but was unable to get any as of now, but is it possible to add a Global Path like we can see in SpringBoot Controllers, where we annotate the class with @RequestMapping and all the functions are preeceded internally by the value mentioned in the parameters of @RequestMapping(value) ?

Something like this :

@Global("user/")
public interface IService{

@Get("details")    ----> Internally would be like "user/details"
public Call<Model> getUserDetails();

@Post("update")    ----> Internally would be like "user/update"
public Call<Model>updateUserDetails(@Body UpdateModel um);

}

**Comment by igor98bn** [comment]
thanks you !
